crossfire/__init__.py

```diff
@@ -43,17 +43,18 @@ def occurrences(
     )
 
 
-def _flatten_dict(data, parent_key, sep="_"):
-    for dict in data:
-        for k, v in dict.items():
-            new_key = parent_key + sep + k if parent_key else k
-            yield new_key, v
-
-
 def flatten(data, nested_columns=None):
     if nested_columns is None:
         nested_columns = NESTED_COLUMNS
-    elif nested_columns not in (NESTED_COLUMNS):
+    nested_columns = set(nested_columns)
```

Comment:
Imaginning that a "regular" user would use `list` of columns to be flatten, I convert it to a `set`. 
Does it make sense?